### PR TITLE
Add promehteus alert rule for missing target

### PR DESCRIPTION
**Pull request by arturo-seijas**
Add promehteus alert rule for missing scrape target

### CODE REVIEW

File: src/prometheus_alert_rules/indico_missing.rule [hunk 1]
```diff
@@ -0,0 +1,8 @@
+alert: IndicoTargetMissing
+expr: up == 0
+for: 0m
+labels:
+  severity: critical
+annotations:
+  summary: Prometheus target missing (instance {{ $labels.instance }})
+  description: "Indico target has disappeared. An exporter might be crashed.\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}"
```
Comment (gregory-schiano) [inline src/prometheus_alert_rules/indico_missing.rule:8]: Should we add an empty line here ?

Comment (arturo-seijas) [reply src/prometheus_alert_rules/indico_missing.rule]: Added

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by nrobinaubertin — APPROVED** [review]
LGTM

**Pull request merged**